pandas/core/indexes/interval.py

```diff
@@ -121,6 +121,9 @@ class IntervalIndex(IntervalMixin, Index):
         Name to be stored in the index.
     copy : boolean, default False
         Copy the meta-data
+    mid
```

Comment:
is this right?

Reply:
Yeah, if it's just the attribute name the attribute docstring is filled in. But currently that's buggy because `@cache_readonly` doesn't copy of the docstring.